CODE_OF_CONDUCT.md

```diff
@@ -88,6 +137,7 @@ The moderators will meet to review the incident and determine:
 We promise to acknowledge receipt within 48 hours (and will aim for a faster response). If this is determined to be an ongoing incident or a threat to physical safety, the working groups' immediate priority will be to protect everyone involved. This means we may delay an "official" response until we believe that the situation has ended and that everyone is physically safe.
```

Comment:
This seems to contradict the 3 day window you described earlier in the doc

Reply:
Yes, this should be edited to 72hrs.